src/test/java/unit/Group.java

```diff
@@ -96,16 +97,39 @@ void listDescribeUsersReflectCreateDelete() throws IOException {
         1, groupDescribed.numMembers, "group # members matches describe output after create");
 
     // `terra group delete --name=$name`
-    TestCommand.runCommandExpectSuccess("group", "delete", "--name=" + name, "--quiet");
+    TestCommand.runCommandExpectSuccess("group", "delete", "--name=" + name1, "--quiet");
 
     // `terra group list`
     groupList =
         TestCommand.runAndParseCommandExpectSuccess(new TypeReference<>() {}, "group", "list");
 
     // check that the group is not included in the list output
     matchedGroup =
-        groupList.stream().filter(listedGroup -> listedGroup.name.equals(name)).findAny();
+        groupList.stream().filter(listedGroup -> listedGroup.name.equals(name1)).findAny();
     assertTrue(matchedGroup.isEmpty(), "group does not appear in list after delete");
+
+    // create another group
+    String name2 = SamGroups.randomGroupName();
+    TestCommand.runAndParseCommandExpectSuccess(
+        UFGroup.class, "group", "create", "--name=" + name2);
+
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list");
+
+    // use regular expression testing the table format and content inside
+    assertNull(cmd.stdErr);
+    assertEquals(0, cmd.exitCode, "group list returned successfully");
+    String[] rows = cmd.stdOut.split("\\r?\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals("EMAIL", rowHead[0].trim().replace("\r", ""));
+    assertEquals("MEMBERS", rowHead[1].trim().replace("\r", ""));
+    assertEquals("POLICIES", rowHead[2].trim().replace("\r", ""));
+
+    for (int i = 1; i < rows.length; i = i + 1) {
+      String[] rowi = rows[i].split("\\s+");
+      assertTrue(rowi[0].matches("^[a-zA-Z0-9_-]+@[a-zA-Z0-9_-]+(\\.[a-zA-Z0-9_-]+)+$"));
+      assertTrue(rowi[1].matches("[0-9]+"));
+      assertTrue(Arrays.asList("[ADMIN]", "[MEMBER]").contains(rowi[2]));
```

Comment:
If there is a member too, it will print like `[ADMIN, MEMBER]` because you're doing `List.toString()`. That's where the square brackets are coming from.

Reply:
done